src/dgc_cert.rs

```diff
@@ -13,19 +13,52 @@ pub struct DgcCertName {
     pub fnt: String,
 }
 
+fn t_empty_if_null<'de, D>(deserializer: D) -> Result<Vec<Test>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let opt = Option::deserialize(deserializer)?;
+    Ok(opt.unwrap_or_default())
+}
+
+fn v_empty_if_null<'de, D>(deserializer: D) -> Result<Vec<Vaccination>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let opt = Option::deserialize(deserializer)?;
+    Ok(opt.unwrap_or_default())
+}
+
+fn r_empty_if_null<'de, D>(deserializer: D) -> Result<Vec<Recovery>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let opt = Option::deserialize(deserializer)?;
+    Ok(opt.unwrap_or_default())
+}
+
```

Comment:
Note that I had to do this because there were 2 tests (originally skipped because of the "kid in unprotected header" problem) that have the following content encoded:

```json
 {
    "r": null,
    "t": null,
    "v":  [
       // ...
    ],
}
```

Instead of having only one value set between `r`, `t`, or `v` they set two of them to `null` and the other to an appropriate array.

Solution based on https://github.com/serde-rs/serde/issues/1098

Reply:
I have generalised this to one generic `empty_if_null` function...

Reply:
FYI: https://docs.rs/serde_with/1.11.0/serde_with/
This can be pretty useful...

Reply:
Nice one. Is there any utility there that would do what we are doing here? I couldn't find one at first glance...

Reply:
In this case I am not sure. However, this a lifesaver to avoid `skip_serialization_if(Option::is_none)` everywhere. For some of my project I discovered this a bit too late... :sweat: